### PR TITLE
Added the Jupyter file for test data generator - Issue 444

### LINKED ISSUE #444: Python - Test data generator (CSV)
# Description

A python script/file which will provide a CSV of generated test data 

## Language

- [ ] Python

## Type of issue

- [ ] Feature

## Checklist:

- **For Feature**

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| test data generator (CSV) | Provide a simple script which can generate test data in CSV file format | feature | I would self-assign this |

**Is your feature request related to a problem? Please describe.**: I spent a while looking for test data generation options that could easily be fitted in to projects and re-used

**Describe the solution you'd like**: I would like to create a useful script which can generate test data in a CSV file, containing multiple options around the included content that the user can add/remove

### PR DESCRIPTION
**Pull request by srimontidutta**
Issue-444
Added the Jupyter file for test data generator of data of Students taking online courses using Faker.
Fixes #444 
Kindly check it.

### CODE REVIEW

**Comment by srimontidutta** [comment]
Also, please change the "medium" tag on this issue to a "level" tag instead.

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
Please Add a README file.

**Comment by srimontidutta** [comment]
Added README file. Please check @vybhav72954

**Comment by vybhav72954** [comment]
Hey @srimontidutta The README you have added, is still not up tp the mark. Kindly see the Markdown part in the [guide](https://github.com/HarshCasper/Rotten-Scripts/blob/master/STYLE_GUIDELINES.md)

File: Python/Test Data Generator/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+faker.Faker() - Creates and initializes a faker generator, which can generate data by accessing properties named after the type of data we want.
+Each call to method fake.name() yields a different (random) result. This is because faker forwards faker.Generator.method_name() calls to faker.Generator.format(method_name).
+
+## Working of Code
+
+This code asks the user's input for the numbers of entries required and generates a fake test dataset comprising of student details enrolled in online courses.
+
+The parameters of the constructed dataset are-
+
+Student ID
```
Comment (vybhav72954) [inline Python/Test Data Generator/README.md:20]: Make this an ordered list

File: Python/Test Data Generator/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Country
+
+Course Comments
+
+## Saving/Exporting Dataset
+
+The generated dataset consisting of the specified number of entries from the user is then exported to the local device in csv format.
+
+
+```python
```
Comment (vybhav72954) [inline Python/Test Data Generator/README.md:45]: Empty Code block?

File: Python/Test Data Generator/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Longitude
+
+Country
+
+Course Comments
+
+## Saving/Exporting Dataset
+
+The generated dataset consisting of the specified number of entries from the user is then exported to the local device in csv format.
```
Comment (vybhav72954) [inline Python/Test Data Generator/README.md:42]: Add CSV in code blocks as well, `csv`

File: Python/Test Data Generator/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## Library used for the generation of fake data
+
+### Faker
+
+Faker is a Python package that generates fake data.
+
+#### Syntax
+
+faker.Faker() - Creates and initializes a faker generator, which can generate data by accessing properties named after the type of data we want.
```
Comment (vybhav72954) [inline Python/Test Data Generator/README.md:11]: Add function names in code blocks

File: Python/Test Data Generator/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+# README
+
+## Library used for the generation of fake data
+
+### Faker
```
Comment (vybhav72954) [inline Python/Test Data Generator/README.md:5]: Don't use a heading rather make this part of an unordered list along with its description

File: Python/Test Data Generator/README.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+# README
+
+## Library used for the generation of fake data
```
Comment (vybhav72954) [inline Python/Test Data Generator/README.md:3]: Rename the heading to `## Dependencies`

File: Python/Test Data Generator/README.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
+# README
```
Comment (vybhav72954) [inline Python/Test Data Generator/README.md:1]: Rename this heading to the name of your script

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
KIndly check the [template](https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md)

Make changes accordingly

**Comment by vybhav72954** [comment]
@srimontidutta Rename the directory you made to `Test_Data_Generator`

**Pull request closed without merging**